pkg/vectorpage/manager.go

```diff
@@ -11,36 +11,44 @@ import (
 type HNSWAdjacencyPage [16][8]uint32
 
 type VectorPageManager struct {
-	btree   *btree.BTree
-	vectors []*hnsw.Point
+	btree *btree.BTree
+	// vectors []*hnsw.Point
 
-	bptree       *bptree.BPTree
-	neighborhood map[hnsw.Id]*hnsw.Friends
+	bptree *bptree.BPTree
+	// neighborhood map[hnsw.Id]*hnsw.Friends
```

Comment:
I can remove the comments, just wanted to show that we're fully encapsulating the `hnsw` inside the `VectorPageManager`